modules/distr-tracing-tempo-config-receiver-tls.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls_{context}"]
+= Configure receiver TLS
+
+The `TempoStack` custom resource (CR) allow to configure TLS on the distributor receivers when is not used with the gateway. You can specify a certificate, first you need to create a Secret with the tls.key and tls.crt keys, Those keys will contains the certificate and the key. Optionally you can specify the CA in a ConfigMap if you want to enable mutual TLS authentication.
```

Comment:
@rubenvp8510, I'm trying to understand this: "when [what?] is not used with the gateway".
WDYT?

Reply:
When tempo stack is not deployed with the gateway in front, Is what I wanted to say :D